running/architecture.rst

```diff
@@ -2,3 +2,72 @@ Architecture
 ============
 
 Deconst is distributed as a set of Docker containers, deployed by an Ansible playbook.
+
+Terminology
+-----------
+
+.. glossary::
+
+  content repository
+    Version controlled repository containing markup that provides content for some subset of the
+    completed site. To begin, we'll assume that all of these are in ``git`` repositories on
+    `GitHub <github.com>`_. Most of our documentation will be in Sphinx reStructuredText, but the
+    developer blog will be in Markdown as part of a Jekyll site.
+
+  map repository
+    Version controlled repository containing plain-text documents that associate subtrees of
+    indexed content, identified by a :term:`content ID` prefix, with subtrees of :term:`paths`
+    on the presented site.
+
+  content ID
+    Unique identifier assigned to a single page of content. By convention, these are URLs that
+    join the https URL of the GitHub repository, ``tree/branch`` if "branch" is not master, and
+    the subpath of the page within the *rendered* output of the documentation build tool. Most of
+    the architecture should treat these as opaque strings. Examples:
+    this page would be ``https://github.com/deconst/deconst-docs/running/architecture`` on master,
+    or ``https://github.com/deconst/deconst-docs/tree/glossary/running/architecture`` until this
+    pull request is merged.
+
+  path
+  paths
+    Subpath within the final presented content of a deconst site. Includes the fully-qualified
+    domain name of the page. For example:
+    ``developer.rackspace.com/sdks/cloud-servers/getting-started/``.
```

Comment:
Because we need to support multiple domains, we need to include the domain. It needs to be unique across the deconst-managed web presence, so that we can map it unambiguously.

I realize this doesn't really make it a "path" any more, but I couldn't think of what a better name should be, without being so generic as to be meaningless.